app/frontend/components/pages/txHistory/transactionHistory.tsx

```diff
@@ -114,13 +120,22 @@ const ExportCSV = ({transactionHistory, stakingHistory}) => {
     .map((header) => `${header}${delimiter}`)
     .join('')
 
-  const transactionTypes = {
-    received: 'Received',
-    sent: 'Sent',
-    rewardAwarded: 'Reward awarded',
+  const enum TransactionType {
+    received = 'Received',
+    sent = 'Sent',
+    rewardAwarded = 'Reward awarded',
+  }
```

Comment:
I would follow the convention and have enum entries with caps, so `RECEIVED = 'Received'`. As mentioned, some of these types might be defined in `explorer-types`. If not, I would rename this to `TxSummaryType` or something compliant with the other types.

Reply:
Refactored